airbyte-integrations/connectors/source-commcare/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.9.15-slim-bullseye as base
```

Comment:
Hold up, Dockerfiles not cool, we're using a thing called baseImage instead, defined in the Metadata. That is probably not relevant to the PR, so just dropping the Dockerfile should be fine.

Reply:
I see, gotcha. I will remove the Dockerfile. 

Just one question, how can do I build the connector image from the base ? once i drop the docker file
I am use to doing this `docker build . -t airbyte/source-commcare:<version>`

Reply:
Bad docs on our side ;-( 

To build a connector locally, you do this: 

`airbyte-ci connectors --name source-WHATEVER build`.

Assuming you have airbyte-ci installed already.

Reply:
Gotcha thank you !!

Any other changes you want me to do here. Have removed the docker file